astropy/units/function/logarithmic.py

```diff
@@ -188,7 +188,7 @@ class LogQuantity(FunctionQuantity):
 
     Parameters
     ----------
-    value : number, `~astropy.units.Quantity`, `~astropy.units.function.logarithmic.LogQuantity`, or sequence of convertible items.
+    value : number or `~astropy.units.Quantity` or `~astropy.units.function.logarithmic.LogQuantity` or or sequence of quantity-like.
```

Comment:
I don't understand the need for this change (note it now has a double "or or")

Reply:
The double "or or" is a mistake, but the change from "," to "or" is just me trying to stick to https://numpydoc.readthedocs.io/en/latest/format.html, which has all options separated by "or". That could be pushed to another PR.

Reply:
Regex finding all ":" + " or " + " or " to replace with ":" + ", " + ", or "

Reply:
✅